frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@dnd-kit/core": "^6.0.5",
     "@dnd-kit/sortable": "^7.0.1",
     "@dnd-kit/utilities": "^3.2.0",
+    "@radix-ui/colors": "^0.1.8",
```

Comment:
Is this changes related to this PR?

Reply:
Yes this is for the color styling that is being used in figma for new designs. This library is used in storybook by Kiran.